data/templates/proxy-ndp/ndppd.conf.tmpl

```diff
@@ -0,0 +1,48 @@
+########################################################
+#
+# autogenerated by nat66.py
+#
+#   The configuration file must define one upstream
+#   interface.
+#
+#   For some services, such as nat66, because it runs
+#    stateless, it needs to rely on NDP Proxy to respond 
+#   to NDP requests.
+#
+#   When using nat66 source rules, NDP Proxy needs 
+#   to be enabled
+#
+########################################################
+
+{% set global = namespace(ndppd_interfaces = [],ndppd_prefixs = []) %}
+
+
+{% if source is defined and source.rule is defined and source.rule is not none %}
+{%   for rule, config in source.rule.items() if config.disable is not defined %}
+{%     if config.outbound_interface is defined %}
+{%         if config.outbound_interface not in global.ndppd_interfaces %}
+{%             set global.ndppd_interfaces = global.ndppd_interfaces + [config.outbound_interface] %}
+{%         endif   %}
+{%         if config.translation.prefix is defined  %}
+{%             set global.ndppd_prefixs = global.ndppd_prefixs + [{'interface':config.outbound_interface,'rule':config.translation.prefix}] %}
+{%         endif    %}
+{%     endif    %}
+{%   endfor %}
+{% endif %}
+
+{% for i in global.ndppd_interfaces %}
+
+proxy {{ i }} {
+    router yes
+    timeout 500
+    ttl 30000
+{%  for map in global.ndppd_prefixs %}
+{%     if map.interface == i %}
+{%         set p = map.rule  %}
+    rule {{ p }} {
```

Comment:
Why not use `map.rule` directly?

Reply:
There is a reason for this. global.ndppd_prefixs contains every rule, and if is needed to filter the rule interface

Reply:
Oh, I see what you mean